build-parent/pom.xml

```diff
@@ -85,7 +85,7 @@
 
     <!-- OpenTracing related versions -->
     <version.opentracing.tracerresolver>0.1.0</version.opentracing.tracerresolver>
-    <version.opentracing.servlet>0.0.9</version.opentracing.servlet>
+    <version.opentracing.servlet>0.1.0-RC1</version.opentracing.servlet>
     <version.opentracing.spanmanager>0.0.5</version.opentracing.spanmanager>
```

Comment:
The `spanmanager` should no longer be required as of 0.30.0 due to the introduction of `ActiveSpan`.